index.js

```diff
@@ -2245,6 +2261,23 @@
         return n < 0 || negativeZero(n) ? Nothing() : slice(0, -n, xs);
       });
 
+  //# append :: a -> [a] -> [a]
+  //.
+  //. Takes a value and a list of values of the same type and appends
+  //. the value to the list.
```

Comment:
Let's mention that the function *returns* a list to reduce the likelihood of a user assuming this modifies the given array. How about the following?

_Returns the result of appending the given value to the given list._

This is less precise than many of the other descriptions, but @joneshf has rightly called me out for writing descriptions which simply restate their corresponding type signatures. For example:

_Takes a value of any type and a list of values of that type, and returns the result of appending the value to the list._